### ISSUE TITLE
Re-enable LED patterns in a separate thread

### ISSUE THREAD

**Issue by gerth2** [RobotCasserole1736/RobotCasserole2020#93]
Add a separate, 200ms-updating thread which calculates what pattern the LED's should have, and sets the output to the PWM controller. 

Shouldn't involve changing much of the existing code, just creating the threading wrapper and moving the update logic to that new thread.

reference [this implementation for info on how to do it.](https://github.com/RobotCasserole1736/RobotCasserole2020/blob/master/RobotCasserole2020/src/main/java/frc/lib/LoadMon/CasseroleRIOLoadMonitor.java#L103)

**Comment by gr1ml0ck01** [comment]
Waiting on Code testing, low priority tho.
